torch/backends/mkldnn/__init__.py

```diff
@@ -86,6 +88,10 @@ def __init__(self, m, name):
     enabled = ContextProp(torch._C._get_mkldnn_enabled, torch._C._set_mkldnn_enabled)
 
 
+if TYPE_CHECKING:
+    enabled: ContextProp
```

Comment:
@int3 re: declare enabled as a global of torch.backends.mkldnn during TYPE_CHECKING

is this what you had in mind?

Reply:
yep!